doc/develop/west/build-flash-debug.rst

```diff
@@ -133,6 +133,32 @@ all the files in the build directory::
 
   west build -t pristine
 
+.. _west-multi-domain-builds:
+
+Multi-domain builds
```

Comment:
I don't think this section placement makes sense, but GitHub won't let me upload the screenshot that shows what I mean. Please check Teams for the screenshot.

Reply:
![image](https://user-images.githubusercontent.com/9802583/183830330-687bc2a4-f983-46bb-be14-c1b6edb34c64.png)

Reply:
Now looking like this:
![image](https://user-images.githubusercontent.com/9802583/184123325-b42fda1e-15fc-465a-beaf-169903f90508.png)

And the `west build` part for sysbuild has been split into a general description, and then a dedicated example.

Hope this is in line with your expectations.